### ISSUE TITLE
A11y_.NETOnlineREPL_.NETEditor_Zoom : After zooming page to 200 %,whole content of .NET Editor is not properly visible.

### ISSUE THREAD

**Issue by pranshu512** [dotnet/try#909]
“[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_.NET Online REPL_Web_Feb2021;#.NET Online REPL;#Win10;#FTP;#DesktopWeb;#WCAG1.4.4;#Zoom;#Chrome;

### **Environment Details:**
Application Name: .NET Online REPL
Operating System: Windows10
Browser Version : Google Chrome Version 87.0.4280.141 (Official Build) (64-bit)

### **Repro Steps:**

1. Open .NET Editor coding screen .
2. Zoom the page to 200%.
3. Check whether whole content of .NET Editor is properly visible or not.

### **Actual:**
After zooming page to 200 %,whole content of .NET Editor is not properly visible.

### **Expected:**
After zooming page to 200 %,whole content of .NET Editor should be properly visible.

### **User Impact:**
Low vision user will not be able to get all content and controls at 200% zoom.

**Comment by fsteffi** [comment]
![image](https://user-images.githubusercontent.com/24580570/203703177-95585520-86dc-4255-b588-889ab8e6c5ec.png)
https://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/hello-world?tutorial-step=1&WT.mc_id=-blog-scottha

@LadyNaggaga  could you please close this bug as issue has been fixed

**Comment by Rama8345** [comment]
**Github Tags:**#Closed;
